### DISCUSSION TITLE
What was the target board for the cifar-10 CNV example notebook?

### DISCUSSION THREAD

**Discussion by AlexMontgomerie** [Xilinx/finn#452] [General]
Hi, for this [example](https://github.com/Xilinx/finn-examples/blob/main/finn_examples/notebooks/1_cifar10_with_cnv_networks.ipynb), I'm curious as to which device was targeted, as I would like to know which device lead to the achieved performance. Thank you!

**Comment by maltanar** [comment]
Hi @AlexMontgomerie , the committed notebook is pre-executed on a Pynq-Z1. Actually the CNV-w1a1 network in `finn-examples` is supported for Pynq-Z1/Z2, Ultra96, ZCU104 and Alveo U250, its folding configuration is the same for all of those platforms and should yield the same performance.

**Reply by AlexMontgomerie** [reply]
Amazing, thanks for letting me know!
